### ISSUE TITLE
[Docs] Update dotnet new templates page

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/uno#14808]
### On which page?

https://platform.uno/docs/articles/get-started-dotnet-new.html

### What's wrong?

At the very least we should update the page to remove .NET 6 and add .NET 8 as the default. But it could be also nice to update it with and extensive table of all the flags, their usage and possible values. 

### Any feedback?

Part of an internal discussion on [Uno Teams](https://teams.microsoft.com/l/message/19:787ad1de1af24252b9041bcbd4ec1ca3@thread.tacv2/1701820908569?tenantId=a297d6c0-b635-41a3-b1e3-558efe71e413&groupId=f55cdec8-036b-4efa-8e1a-c4129ba9bf12&parentMessageId=1701820908569&teamName=Uno%20Platform&channelName=Docs%20and%20Getting%20Started&createdTime=1701820908569).

**Comment by Youssef1313** [comment]
Removing .NET 6 was already done in https://github.com/unoplatform/uno/pull/14776 except for `unoapp-uwp-net6` which is waiting on https://github.com/unoplatform/uno/issues/14775

**Comment by cconner100** [comment]
Will this include creating a new project that allows for Centralized nuget configurations?  Also using the variable for the .net version?

**Comment by MartinZikmund** [comment]
Should be up to date now
